t/t1092-sparse-checkout-compatibility.sh

```diff
@@ -622,14 +623,7 @@ test_expect_failure 'sparse-index is not expanded (with ignored files outside co
 	git -C sparse-index reset --hard &&
 	ensure_not_expanded checkout rename-out-to-out -- deep/deeper1 &&
 	git -C sparse-index reset --hard &&
-	ensure_not_expanded restore -s rename-out-to-out -- deep/deeper1 &&
-
-	echo >>sparse-index/README.md &&
-	ensure_not_expanded add -A &&
-	echo >>sparse-index/extra.txt &&
-	ensure_not_expanded add extra.txt &&
-	echo >>sparse-index/untracked.txt &&
-	ensure_not_expanded add .
+	ensure_not_expanded restore -s rename-out-to-out -- deep/deeper1
```

Comment:
Should this hunk be squashed into the commit that introduced the changed code?

Reply:
This is an accident of a cherry-pick from a different place. Will fix.